SETUP.md

```diff
@@ -0,0 +1,78 @@
+    'knitcitations', 'knitr', 'learnr', 'magick', 'nomnoml', 'officedown', 'officer', 'pagedown',
+    'pander', 'pixiedust', 'pkgdown', 'printr', 'r2d3', 'reactable', 'reticulate', 'revealjs', 'rgl',
+    'rhandsontable', 'rmarkdown', 'rmdformats', 'roxygen2', 'rsconnect', 'rticles', 'sass', 'spelling', 
+    'stargazer', 'styler', 'svglite', 'tables', 'tangram', 'tikzDevice', 'tinytex', 'tufte', 'usethis',
+    'webshot', 'workflowr', 'xaringan', 'xfun', 'xtable', 'yaml', 'ztable')
+```
+
+You may install packages with `install.packages()` or with the **pak** package's `pkg_install()` function, i.e.:
+
+```
+pak::pkg_install(reqd_pkgs)
+```
+
+Note the `pkg_install()` will only install missing packages (and not update current packages) unless you set `upgrade = TRUE`. It is generally a good idea to work with the latest version of packages when preparing a contribution or, at minimum if you receive errors while building, upgrading your packages is a good first step in debugging.
+
+### LaTeX Packages
+
+Due to the cookbook's discussion of various [graphical devices](https://bookdown.org/yihui/rmarkdown-cookbook/graphical-device.html), a number of LaTeX packages are also required. Some may already be installed in `tinytex` depending on what version of the package you have.
+
+If you have never used LaTeX or `tinytex` before, first run:
+
+```
+tinytex::install_tinytex()
+```
+
+To check if you have the needed LaTeX packages, run:
+
+```
+tex_pkgs <- c('pgf', 'preview', 'xcolor', 'grfext')
+setNames(tex_pkgs %in% tinytex::tl_pkgs(), pkgs)
+```
+
+If the results for any of the preceding packages is `FALSE`, you may install it with `tinytex::tlmgr_install()`. 
+
+More about `tinytex` and troubleshooting missing LaTeX packages can be found in the cookbook [here](https://bookdown.org/yihui/rmarkdown-cookbook/install-latex-pkgs.html). 
+
+### Other System Dependencies
+
+For Windows users, the section of the book on the [Asymptote language engine](https://bookdown.org/yihui/rmarkdown-cookbook/eng-asy.html) does not build properly so it will be skipped. For users with other operating systems, this section will be skipped if Asymptote is not installed on your computer. If they wish to build this section, they must download and install the [Asymptote software](https://asymptote.sourceforge.io/) from its website. After you install it, you may confirm that it is on your system's path by running the terminal command `where asy`.
+
+## Building the Book
+
+Once you have set up all needed dependencies, there are multiple ways to build the book. 
+
+If you currently use `make` (or set it up following [Software Carpentry's instructions](https://swcarpentry.github.io/make-novice/)), you may type `make gitbook` in the terminal. 
+
+Otherwise, you may preview the book by running `bookdown::render_book()` in the R console or, if you are using RStudio, clicking on `Addins > Preview Book`. 
```

Comment:
Good point -- sorry, I will clarify the different use cases (testing locally vs rendering for actual site.)

To clarify my understanding, I have a Build All button instead of Build Book because the Build type in the `Rproj` is set to 'Makefile'. I know we can get the Build Book button if we change this to 'Website', but then I believe that it will also not use `_render.R`. Is that correct? (The reason I ask is that Build All does not let one chose to just build one type of output. So, unfortunately, it does also build a "fake" PDF -- not the actual one, but it still makes it slower than necessary.)